go.mod

```diff
@@ -53,7 +53,7 @@ require (
 	github.com/google/btree v1.1.2
 	github.com/google/pprof v0.0.0-20211122183932-1daafda22083
 	github.com/google/skylark v0.0.0-20181101142754-a5f7082aabed
-	github.com/google/uuid v1.3.1
+	github.com/google/uuid v1.4.0
```

Comment:
Why upgrade libraries?

Reply:
The imported package by kvproto imports uuid v1.4.0

Reply:
It seems `go get -u github.com/pingcap/kvproto` will also upgrade its deps. I will uses `go get github.com/pingcap/kvproto` instead.